madgraph/various/misc.py

```diff
@@ -1001,7 +1001,19 @@ def call_stdout(arg, *args, **opt):
 def copytree(src, dst, symlinks = False, ignore = None):
   if not os.path.exists(dst):
     os.makedirs(dst)
-    shutil.copystat(src, dst)
+    try:
+      shutil.copystat(src, dst)
+    except PermissionError:
+        if os.path.realpath(src).startswith('/cvmfs') and os.path.realpath(dst).startswith('/afs'):
+           # allowing missmatch from cvmfs to afs since sounds to not create issue --at least in general-- 
+           logger.critical('Ignoring that we could not copy permissions from %s to %s', src, dst)
```

Comment:
```suggestion
           logger.critical(f'Ignoring that we could not copy permissions from {src} to {dst}', src, dst)
```
Not needed, but for consistency with the suggestion below